core/src/main/kotlin/behavior/interaction/AutoCompleteInteractionBehavior.kt

```diff
@@ -28,9 +28,9 @@ public interface AutoCompleteInteractionBehavior : DataInteractionBehavior {
  *
  * The provided choices are only suggestions and the user can provide any other input as well.
  *
- * @see IntChoiceBuilder
+ * @see IntegerOptionBuilder
  */
-public suspend inline fun AutoCompleteInteractionBehavior.suggestInt(builder: IntChoiceBuilder.() -> Unit) {
+public suspend inline fun AutoCompleteInteractionBehavior.suggestInt(builder: IntegerOptionBuilder.() -> Unit) {
```

Comment:
this will also allow specifying min and max values (but they aren't used so it's probably fine)

Reply:
Don't they allow defining ranges?

Reply:
AutoComplete suggestions provide a list of selectable inputs, so there is no range i guess

Reply:
should their be a BaseChoiceBuilder type for this then?

Reply:
IntegerOptionBuilder is used anyways, the only way would be to make 2 IntegerOptionBuilder and 2 NumberOptionBuilder, which seems a bit unnecessary

Reply:
what if we introduce an interface like, `RangedNumericOptionBuilder` and extends them; that only contains the min and max values
we won't introduce a new Double and Int values; instead we just use the one without values (interface) on the autocomplete functions and apply it to the one with ranges (complete type)

Reply:
That interfae is exactly what the NumericOptionBuilder is right now

the issue is that the int option builder is the same as the int choice builder

Reply:
Hmmm, we could go with this for now? (current implementation)
so we would need an additonal interface?

Reply:
We would need an additional class the interfaces are fine, but I doubt this will be a huge issue for now

Reply:
You are right, we can figure this out on the go